README.md

```diff
@@ -11,9 +11,80 @@
   </a>
 </p>
 
-This Maven plugin is a one-stop-shop for building and deploying Java applications for Docker, Kubernetes and OpenShift.
+### Introduction
+This Maven plugin is a one-stop-shop for building and deploying Java applications for Docker, Kubernetes and OpenShift. It brings your Java applications on to Kubernetes and OpenShift. It provides a tight integration into maven and benefits from the build configuration already provided. It focuses on two tasks:
++ Building Docker images
++ Creating OpenShift and Kubernetes resources
 
-What to get started fast? Check out the [busy Java developers guide to developing microservices on Kubernetes and Docker](https://blog.fabric8.io/a-busy-java-developers-guide-to-developing-microservices-on-kubernetes-and-docker-98b7b9816fdf).
+### Usage
+To enable fabric8 maven plugin on your project just add this to the plugins sections of your pom.xml: 
+
+```
+      <plugin>
+        <groupId>io.fabric8</groupId>
+        <artifactId>fabric8-maven-plugin</artifactId>
+        <version>${fmp.version}</version>
```

Comment:
It's better to put any working version number. Make a note of version number and point maven central link for plugin.